docs/platform/api/nodebalancers.md

```diff
@@ -0,0 +1,166 @@
+  - '[Getting Started with NodeBalancers](https://linode.com/docs/platform/nodebalancer/getting-started-with-nodebalancers/)'
+---
+
+[NodeBalancers](https://www.linode.com/nodebalancers) can be used to provide high availability load balancing for almost any type of website or service hosted on a Linode. This guide will demonstrate how to use the Linode API to create a NodeBalancer with two back end nodes.
+
+You will need an access token for the Linode API to complete the steps in this guide. The easiest way to get an access token is by visiting the beta [Linode Manager](https://cloud.linode.com/profile/tokens); for more information, as well as a basic introduction to using the Linode API, see our guide on [Getting Started with the Linode API](/docs/platform/api/getting-started-linode-api).
+
+## Create a NodeBalancer
+
+1.  Using a text editor, create a file to store configuration options:
+
+    {{< file "nodebalancer.json" json >}}
+{
+  "region": "us-central",
+  "label": "nodebalancer-1",
+  "client_conn_throttle": 10
+}
+{{< /file >}}
+
+2.  Create a NodeBalancer by making a POST request to the `/nodebalancers` endpoint:
+
+        curl https://api.linode.com/v4/nodebalancers \
+        -H "Authorization: Bearer $TOKEN" \
+        -H "Content-Type: application/json" \
+        -X POST -d @nodebalancer.json
+
+3.  If the NodeBalancer is successfully created, the response will include its ID. Copy the ID to use in subsequent requests.
+
+## Add Configuration
+
+NodeBalancers are created without a configuration profile attached, so you will need to add one.
+
+1.  Create a new configuration file:
+
+    {{< file "nodebalancer-config.json" >}}
+{
+  "label": "nodebalancer-1",
+  "port": 80,
+  "check": "connection"
+  }
+{{< /file >}}
+
+2.  Substitute the NodeBalancer's ID into the URL below:
+
+        curl https://api.linode.com/v4/nodebalancers/$nodebalancer-id/configs \
+        -H "Authorization: Bearer $TOKEN" \
+        -H "Content-Type: application/json" \
+        -X POST -d @nodebalancer-config.json
+
+{{< note >}}
+For more information about configuration options, see our [NodeBalancer Reference Guide](https://www.linode.com/docs/platform/nodebalancer/nodebalancer-reference-guide/)
+{{< /note >}}
+
+## Add Nodes
+
+Even with a working configuration profile, the NodeBalancer isn't doing anything yet, since it has no nodes connected to it. Repeat the steps in this section for each node you would like to add; usually you will want at least two nodes so that the NodeBalancer can balance the load between the two.
+
+### Create Linodes
+
+1.  Add the following options to a new config file. Adjust the type, image, and region to suit your needs; make sure the new Linode is in the same region as your NodeBalancer and choose a secure root password.
+
+    {{< file "create-linode.json" json >}}
+{
+  "region": "us-central",
+  "type": "g5-standard-2",
+  "image": "linode/debian9",
+  "root_pass": "password"
+}
+{{< /file >}}
```

Comment:
This request will create and immediately boot the Linode.  I believe that means that you'll need to log into the box and bring up the private IP manually after you've added it below.  If you don't want to do this, you could pass `"booted": false` into this request, then boot the Linode after the private IP has been added.  We have "Create a Linode with Private Networking" on our roadmap, so hopefully this is a temporary concern.